### ISSUE TITLE
[Group Request] Cloud Native Up State NY

### ISSUE THREAD

**Issue by halil-bugol** [cncf/communitygroups#73]
### City or location name for your CNCG

Cloud Native Up State NY

### What country do you want to start your community group in?

United States

### Tell us about yourself. List your name, email, company, GitHub URL, LinkedIn URL, and CNCF Slack workspace name (or if you need to be invited), and a required profile link on community.cncf.io

Halil Bugol, Co-Founder at Kubezy, CNCF Ambassador, https://www.linkedin.com/in/halilbugol/, CNCF Slack: Halil

### Tell us about your co-organizer. List their name, email, company, GitHub URL, LinkedIn URL, and CNCF Slack workspace name (or if they need to be invited), and a required profile link on community.cncf.io

Stephane Yasar, Co-Founder at Kubezy, stephane@kubezy.com, https://www.linkedin.com/in/st%C3%A9phane-y-828a89233/, CNCF Slack: Stephane-Yasar
Yavuz Selim Gul, Business Executive at Kubezy, yavuz@kubezy.com, https://www.linkedin.com/in/yavuzselimgul/
(Chapter Leader) Hakan Gul, Product Manager at Kubezy, hakan@kubezy.com, https://www.linkedin.com/in/hakanahmetgul/ https://community.cncf.io/u/mzwvap/ CNCF Slack: Hakan

### If you already run a meetup, please link it here

https://community.cncf.io/istanbul/

### What is your mission statement or objective for this new chapter?

This region and the region covering the Southern parts of Canada look quite empty. Our company has an established office here. If we can establish a CNCG in this region, we can easily support it both financially and in other ways. I will host the same series of University Laboratories, Startup programs and other events here as I aimed in Istanbul.

### Code of Conduct

- [X] I/We agree to follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)

### Where organizers reside

- [X] I/We confirm I/we am/are local to or live near the city of the above-proposed chapter.

### Organizers companies/affiliations

- [ ] I/We agree if there is more than one organizer now or onboarded in the future, the majority come from different companies

### Inclusive

- [X] I/We agree. To assure this, I/we have recently completed the [Inclusive Open Source Community Orientation (LFC102) course](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)

### Vendor Neutral

- [X] I/We have read what [vendor-nuetrality means](https://contribute.cncf.io/maintainers/community/vendor-neutrality/), and agree that the main content of our meetups will remain vendor-nuetral

### Agree to use CNCG branding

- [X] I/We agree to use the CNCG logo in my/our chapter and social branding

### Registration through CNCF platforms

- [X] I/We agree to host free registration in the main CNCF platform

### Remain an active chapter

- [X] I/We agree to have a meetup at least once every 90 days, or else communicate reasoning for upcoming inactivity.

### Collaboration with other chapters

- [X] I/We acknowledge the CNCF right to approve additional chapters in my area based on this criteria

### Aim to have a diverse representation of speakers in your meetups

- [X] I/We will do our best to host diverse speakers throughout the year

**Comment by hakanagul** [comment]
This will be great for the region. I have really high hopes and I believe we can achieve a lot

**Comment by abebars** [comment]
This is amazing and I am super excited to see that Cloud Native York is Expanding and Growing. Kudos @halil-bugol and Good Luck

**Comment by k8tgreenley** [comment]
@halil-bugol can you please answer the below questions:

- Confirm your involvement since you aren't currently located in New York? Who from the list of organizers is local to upstate NY?
- Where in upstate New York is this taking place?
- Are the organizers only from Kubezy?
- Can you please provide the cadence of your meetup, i.e. will it be monthly, quarterly, etc.? What day of the week will it be held?
- Is there a specific audience you are trying to reach?

**Comment by AudMonte01** [comment]
Thank you, @halil-bugol - do you have anyone who can commit to co-organizing with you that lives in upstate New York full time?

**Comment by AudMonte01** [comment]
Thank you for confirming, @hakanagul

Each one of you will need to confirm on GitHub in this ticket that you agree to the above organizer terms, and have taken the Open Source Inclusive Training, mentioned.

**Comment by hakanagul** [comment]
@AudMonte01 I finished the Open Source Inclusive Training and I agree to the terms above.

**Comment by stephaneyasar** [comment]
@AudMonte01  I agree to all the terms above.

**Comment by yavuzselimgul** [comment]
I have successfully completed the Open Source Inclusive Training and hereby agree to the terms outlined above.

**Comment by AudMonte01** [comment]
Thanks, @halil-bugol Halil - there is just one problem. All the organizers are from the same company. The third check-box was originally marked, "I/We agree if there is more than one organizer now or onboarded in the future, the majority come from different companies", but I had to un-check it.

That said, please let me know if there is one of you keen to organize, and have others in mind that can help that are from other companies.

**Comment by AudMonte01** [comment]
Yes, you can add an organizer who lives in NY part time. But given there would be 5 organizers with 20% being that of a different company, we should aim for more than 50% from different companies - "the majority". So you would have to add 2 more organizers from different companies.

**Comment by AudMonte01** [comment]
Any word on this @halil-bugol ?

**Comment by AudMonte01** [comment]
Hello @halil-bugol and @yavuzselimgul - how is the search going?

**Comment by halil-bugol** [comment]
@AudMonte01 We found 3 more organizers from AWS and other companies. We will talk to them and add them before the end of this month.

**Comment by AudMonte01** [comment]
This issue has been open for over 6 months. Please re-open it when your team is properly assembled meeting all the perimeters.

Kind regards,
Audra

**Comment by halil-bugol** [comment]
Hi @newgoliath 

The team here is not available to do this task right now. @AudMonte01 I think this issue should be closed and a new issue should be opened for this chapter.

@newgoliath good luck!

**Comment by hakanagul** [comment]
Hey @newgoliath @AudMonte01 

I would still like to be a part of this. I am no longer affiliated with Halil -- however, I think this is still very valuable and I can contribute. I just finished my 3rd year at Rochester Institute of Technology.

My personal email is hag4126@rit.edu, and my linkedin is [hakanahmetgul](https://www.linkedin.com/in/hakanahmetgul)

Looking forward to hear more!

**Comment by AudMonte01** [comment]
Thank you for your interest @hakanagul and @halil-bugol for the updates. I agree that a new issue should be generated with all that are interested.

@hakanagul - please convene with @cfilleke and @shpwrck to come to a common working and commitment agreement with taking the organizer standards into consideration.

https://github.com/cncf/communitygroups/blob/main/organizer_roles.md

This issue is now closed.
